source/_patterns/02-molecules/leads/lead-gallery.twig

```diff
@@ -2,9 +2,18 @@
 
   <div class="c-lead-gallery__top">
 
-    {% include '@molecules/article/figure-with-caption.twig' %}
+    <div class="c-lead-gallery__image js-toggle js-gallery-toggle" data-toggled="c-main" data-prefix="gallery">
+      {%
+        include 'molecules-figure'
+        with {
+          caption: false,
+          credit: false
```

Comment:
instead of specifying `credit` as `false,` could you fill in a `bottom_caption` block in this template?

Reply:
Yup, I think this is what I'll do.